src/lib/collab/schema.js

```diff
@@ -445,7 +513,7 @@ export const nodes = [
     {
       selectable: false,
       content:
-        '(paragraph | panel | blockquote | orderedList | bulletList | rule | heading | codeBlock | unsupportedBlock)+',
+        '(paragraph | panel | blockquote | orderedList | bulletList | rule | heading | codeBlock | mediaSingle | unsupportedBlock)+',
```

Comment:
@nono are you aware of this new schéma?

Reply:
It's on my todo-list to update cozy-stack with the new schema for markdown generation